### DISCUSSION TITLE
Does the app allow auto-emailing links?

### DISCUSSION THREAD

**Discussion by sirken** [pglombardo/PasswordPusher#832] [Q&A]
Hi everyone. Does the option exist to automatically notify via email when a new link is generated? I looked at the SMTP settings in the docs, but these appear to be for account-related functionality only.

The idea behind this would be for an internal deployment, where all generated links go to the same inbox.

Thank you!

**Comment by pglombardo** [comment]
Hi @sirken this option doesn't exist currently.  What would you use that feature for?  Auditing of the instance?  e.g. what are you trying to solve?

Let me know - maybe I can add the functionality.

**Comment by sirken** [comment]
Hi @pglombardo.

Yes, this option could be helpful for auditing.

More specifically for our situation, looking to use it for an internal production deployment where all app submissions will be emailed to a single department or individual. It ends up being less of an anonymous credential sharing portal, and more of a direct submission tool.

![image](https://user-images.githubusercontent.com/8664158/219455676-1e982c8b-db1f-425a-8b9d-8bd185ffc36d.png)

## Current 3-step workflow
- User generates link -> user copies link -> user sends email to admin

## Optimal 1-step workflow
- User generates link -> app auto-emails link to address(s) defined in config, removing the manual steps of copy/paste & email

## Simple idea
`PWP__SEND_TO_EMAIL`='user@domain.com'
  - If SMTP settings are set, all new submissions will be emailed here

## More flexible idea
`PWP__SEND_TO_EMAIL`='user@domain.com'

`PWP__DISABLE_EMAIL_CHECKBOX`='true'
  - If `PWP__SEND_TO_EMAIL` is defined, show this checkbox option on the webpage, something like:
    - ☑ Submit submission automatically. This will be emailed directly when it is generated.
  - If set to `true`, checkbox is grayed out and not editable but still visible
 ![image](https://user-images.githubusercontent.com/8664158/219464279-a5a02b76-1843-4c19-a2bd-067abefb387f.png)
  - If set to `false`, the checkbox can be unchecked, and email will not be sent
![image](https://user-images.githubusercontent.com/8664158/219465085-669d9e5c-604e-46e5-84f5-c5ad77d08277.png)
